### PR TITLE
update hxformat

### PR DESCRIPTION
**Pull request by nahiassis**

### CODE REVIEW

**Comment by ShadowMario** [comment]
What is this supposed to do

**Comment by Steviegt6** [comment]
> i found this in the haxe library for lime it is something for the haxe files

You made this exact same PR on the actual `Funkin` repo, are you just fishing for commits?

File: hxformat.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
 		}
 	},
 	"sameLine": {
+		"ifBody": "same",
 		"ifElse": "next",
```
Comment (Steviegt6) [inline hxformat.json:12]: Also, wouldn't these conflict in styles? LMAO

**Comment by nintendofan44** [comment]
no
just tryna help
but if you dont want it
no problem at all

File: hxformat.json (hunk 1 above)
Comment (nintendofan44) [reply hxformat.json:12]: something on the haxe library for lime
image

Comment (Steviegt6) [reply hxformat.json:12]: > 
> 
> something on the haxe library for lime image

This isn't the Haxe library for Lime, though?

Comment (nintendofan44) [reply hxformat.json:12]: but its basically the same thing
same file name,
same format,
so i wanted to update it here too

**Comment by ShadowMario** [comment]
"Something for the haxe files"
Oh wow! That helps so much. Seriously, what does this even do??

**Comment by KlavierGayming** [comment]
got the exact same question

**Comment by nintendofan44** [comment]
https://github.com/haxelime/lime/blob/develop/hxformat.json

it's on here

it's is a file linked (i believe) to how the haxe files are formatted

**Comment by Micasddsa** [comment]
So this updates the format but unless this actually fixes a compiling/code issue, there isn't a need for this.
The last time I checked, the game worked fine as is without this.

(But then again, this is coming from the official repo of lime so idk if it's necessary or not or whatever.)

**Pull request closed without merging**